src/trackinfoobject.cpp

```diff
@@ -783,13 +783,14 @@ bool TrackInfoObject::isDirty() {
 
 int TrackInfoObject::getRating() const {
     QMutexLocker lock(&m_qMutex);
-    return m_iRating;
+    return m_metadata.getRating();
```

Comment:
I think, display of the email address is unnecessary in this case. With Windows media player, and some taggers, they don't even save a email address in that field, but a string about the app. Windows media player puts "Windows Media 9 Series rating=128 counter=0" for its 3 star rating.

I need some traktor rated files, anyone able to provide?